05-Organizing-and-Executing-Simulations/01-Organizing-Data.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>Move the job document definition to here?</p><p> </p><p>"The <strong>job document</strong> is a persistent dictionary where you can record the job's status."</p><p>...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done.</p>